bindings/java/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+add_subdirectory(com/google/iree/native)
```

Comment:
Would it make more sense to have each level just use `iree_add_all_subdirs()` rather than this explicitly reaching down multiple levels?

Reply:
Java convention is to have a directory structure matching the package structure, and to have package structure of the format `java.com.[company name].[project].[foo]`. 

If the intermediate folders only contain a single subfolder, GitHub (and some other tools) will collapse them automatically:

![image](https://user-images.githubusercontent.com/4010439/96651082-e80fe300-12e8-11eb-9833-fb36c1edcfd2.png)

Until we actually put something useful in the middle directories, keeping them empty and reaching down from `bindings/java` was my suggestion.

Reply:
Yeah I know about the java package convention, but good point about the advantages of keeping them empty :-)